examples/easit-oauth-integration/EasitTestOauthDatastore.js

```diff
@@ -77,6 +77,14 @@ fluid.defaults("gpii.oauth2.easitSampleDataStore", {
                 oauth2ClientSecret: false,
                 redirectUri: false,
                 allowDirectGpiiTokenAccess: false
+            },
+            {
+                id: 6,
+                name: "First Discovery",
+                oauth2ClientId: "client_first_discovery",
```

Comment:
@acheetham, @jobara, any ideas of a sensible GPII solution id for the first discovery tool? "org.a11y.tools.firstDiscovery"?